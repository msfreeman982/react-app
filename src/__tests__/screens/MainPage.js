"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
const react_1 = __importDefault(require("react"));
const react_test_renderer_1 = __importDefault(require("react-test-renderer"));
const MainPage_1 = __importDefault(require("../../Components/Pages/MainPage/MainPage"));
it('renders correctly', () => {
    const tree = react_test_renderer_1.default.create(react_1.default.createElement(MainPage_1.default, null)).toJSON();
    expect(tree).toMatchSnapshot();
});
