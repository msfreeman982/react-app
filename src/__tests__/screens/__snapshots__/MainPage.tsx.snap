// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="main"
  >
    <div
      className="main__heroDescriptionContainer"
    >
      <h2>
        Choose hero
      </h2>
    </div>
    <div>
      <h2>
        Marvel Heroes
      </h2>
      <button
        className="main__buttonMore"
        onClick={[Function]}
      >
        Show more
      </button>
    </div>
  </div>
  <div
    className="loader "
  >
    <svg>
      marvel-logo.svg
    </svg>
  </div>
</div>
`;
